Configurations/10-main.conf

```diff
@@ -1345,6 +1345,7 @@ sub vms_info {
             my @ex_libs = ();
             push @ex_libs, 'ws2_32.lib' unless $disabled{sock};
             push @ex_libs, 'gdi32.lib advapi32.lib crypt32.lib user32.lib';
+            push @ex_libs, 'ncrypt.lib';
```

Comment:
It's tricky one. Formally there is no reason to link *everything* with ncrypt.lib. Well, of course in static build ncrypt.lib needs to be there for every .exe that pulls in e_cng.obj. But in shared build cng.dll would be the only one that needs to be linked with it. MSC has solution for that, and it's #pragma comment(lib,"ncrypt") that can be added directly to e_cng.c. And it's going to work out in both static and dynamic builds. [The pragma is tolerated by mingw compilers, but does nothing.] Another argument against adding it as above is that people still seem to be asking about compiling for legacy version, and there is no ncrypt.lib there. Even in such case above mentioned #pragma would do the trick, because it can be conditional...